terraso_backend/apps/soil_id/models/depth_dependent_soil_data.py

```diff
@@ -13,13 +13,273 @@
+PH_TESTING_SOIL_WATER_1_5 = "1:5 soil/water"
+PH_TESTING_SOIL_CACL2_1_1 = "1:1 soil/0.1 M CaCL2"
+PH_TESTING_SOIL_CACL2_1_2 = "1:2 soil/0.1 M CaCL2"
+PH_TESTING_SOIL_CACL2_1_5 = "1:5 soil/0.1 M CaCL2"
+PH_TESTING_SOIL_KCL_1_1 = "1:1 soil/1.0 M KCL"
+PH_TESTING_SOIL_KCL_1_2_5 = "1:2.5 soil/1.0 M KCL"
+PH_TESTING_SOIL_KCL_1_5 = "1:5 soil/1.0 M KCL"
+PH_TESTING_SATURATED_PASTE_EXTRACT = "saturated paste extract"
+
+PH_TESTING_SOLUTIONS = (
+    (PH_TESTING_SOIL_WATER_1_1, _("1:1 soil/water")),
+    (PH_TESTING_SOIL_WATER_1_2, _("1:2 soil/water")),
+    (PH_TESTING_SOIL_WATER_1_2_5, _("1:2.5 soil/water")),
+    (PH_TESTING_SOIL_WATER_1_5, _("1:5 soil/water")),
+    (PH_TESTING_SOIL_CACL2_1_1, _("1:1 soil/0.1 M CaCL2")),
+    (PH_TESTING_SOIL_CACL2_1_2, _("1:2 soil/0.1 M CaCL2")),
+    (PH_TESTING_SOIL_CACL2_1_5, _("1:5 soil/0.1 M CaCL2")),
+    (PH_TESTING_SOIL_KCL_1_1, _("1:1 soil/1.0 M KCL")),
+    (PH_TESTING_SOIL_KCL_1_2_5, _("1:2.5 soil/1.0 M KCL")),
+    (PH_TESTING_SOIL_KCL_1_5, _("1:5 soil/1.0 M KCL")),
+    (PH_TESTING_SATURATED_PASTE_EXTRACT, _("Saturated Paste Extract")),
+    (OTHER, _("Other")),
+)
+
+
+PH_TESTING_INDICATOR_STRIP = "pH indicator strip"
+PH_TESTING_INDICATOR_SOLUTION = "pH indicator solution"
+PH_TESTING_METER = "pH meter"
+
+PH_TESTING_METHODS = (
+    (PH_TESTING_INDICATOR_STRIP, _("pH indicator strip")),
+    (PH_TESTING_INDICATOR_SOLUTION, _("pH indicator solution")),
+    (PH_TESTING_METER, _("pH meter")),
+    (OTHER, _("other")),
+)
+
+SOIL_TESTING_DRY_COMBUSTION = "Dry combustion"
+SOIL_TESTING_WET_OXIDATION = "Wet oxidation (Walkey-Black)"
+SOIL_TESTING_LOSS_ON_IGNITION = "Loss-on-ignition"
+SOIL_TESTING_REFLECTANCE_SPECTROSCOPY = "Reflectance spectroscopy"
+SOIL_TESTING_FIELD_REFLECTOMETER = "Field reflectometer"
+
+SOIL_TESTING_METHODS = (
+    (SOIL_TESTING_DRY_COMBUSTION, _("Dry combustion")),
+    (SOIL_TESTING_WET_OXIDATION, _("Wet oxidation (Walkey-Black)")),
+    (SOIL_TESTING_LOSS_ON_IGNITION, _("Loss-on-ignition")),
+    (SOIL_TESTING_REFLECTANCE_SPECTROSCOPY, _("Reflectance spectroscopy")),
+    (SOIL_TESTING_FIELD_REFLECTOMETER, _("Field reflectometer")),
+    (OTHER, _("Other")),
+)
+
 
 class DepthDependentSoilData(BaseModel):
     soil_data_input = models.ForeignKey(SoilData, on_delete=models.CASCADE)
     depth_top = models.IntegerField(blank=True)
     depth_bottom = models.IntegerField(blank=True)
+    texture = models.CharField(null=True, choices=TEXTURES)
+    rock_fragment_volume = models.CharField(null=True, choices=ROCK_FRAGMENT_VOLUMES)
+    color_hue = models.CharField(null=True, choices=COLOR_HUES)
+    color_hue_substep = models.CharField(null=True, choices=COLOR_HUES_SUBSTEPS)
+    color_value = models.CharField(null=True, choices=COLOR_VALUES)
+    color_chroma = models.CharField(null=True, choices=COLOR_CHROMAS)
+    conductivity = models.DecimalField(
+        null=True, max_digits=100, decimal_places=2, validators=[MinValueValidator(0)]
+    )  ## max digit dont know
```

Comment:
@shrouxm Is there a default max_digit I should use? It wasn't specified in the specs

Reply:
you can leave `max_digits` out if you don't need it! if you have a `MaxValueValidator` on it, or if there's just no specification that the value should have an upper bound then you can just not supply the `max_digits` arg to `DecimalField`

Reply:
@shrouxm I think max_digits are required for DecimalField; I get this error if I don't have one: `soil_id.DepthDependentSoilData.soil_organic_matter: (fields.E132) DecimalFields must define a 'max_digits' attribute.`, even when I define it as none